opentracing-metrics-prometheus-spring-autoconfigure/pom.xml

```diff
@@ -38,6 +38,12 @@
     </dependency>
 
     <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-spring-web-autoconfigure</artifactId>
+      <scope>provided</scope>
```

Comment:
Use of the metrics auto config is not dependent upon using spring web - so would prefer not to pull it in unnecessarily - however if the app is using it, then the conditional auto config will kick in.